cgd_tx_eff/tfx_cgd.xml

```diff
@@ -42,7 +42,7 @@
  
   <inputs>
     <param name="in_vcf" format="vcf" optional="false" type="data" label="Input VCF" />    
-    <param name="ccds_map" format="csv,gff" optional="false" type="data" label="Input CSV or GFF with RefSeq-to-CCDS mappings. Use refseq_to_ccds.py to convert GFF to CSV."/>
+    <param name="ccds_map" format="csv,gff" optional="false" type="data" label="Input CSV with RefSeq-to-CCDS mappings."/>
```

Comment:
Seems like you've removed language that suggests we can pass in a GFF as an argument, but the format on this line still claims it accepts `csv,gff` ?

Reply:
Good catch.  And the ``bcbio-gff`` has gone away.